crates/hc_bundle/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- Refactor: rename UID to network seed [\#1493](https://github.com/holochain/holochain/pull/1493)
```

Comment:
The changelog could be more exacting and technical in communicating these changes and that would be helpful for developers.

Like: 
Breaking: The property `integrity.uid` of DNA Yaml files is now specified as `integrity.network_seed`. Functionality has not changed, only the property name has.

Reply:
This is a breaking change since backwards compatibility is not introduced

Reply:
@Connoropolous Thanks for the suggestion. I realized it's a breaking change and hadn't merge it yet therefore. Updated